atom/index.d.ts

```diff
@@ -95,6 +95,14 @@ export interface WritableAtom<Value = any> extends ReadableAtom<Value> {
    * @param newValue New store value.
    */
   set(newValue: Value): void
+
+  /**
+   * Low-level method to read store’s value without calling `onStart`.
+   *
+   * Try to use only {@link ReadableAtom#get}.
+   * Without subscribers, value can de undefined.
+   */
+  value: undefined | Value
```

Comment:
We already have it in `ReadableAtom`. Why do we need it here too?

Reply:
That's strange, I have no idea how it got here. Will ✂️ it